api/src/main/java/net/luckperms/api/model/data/NodeMap.java

```diff
@@ -109,6 +109,25 @@
      */
     DataMutateResult.@NonNull WithMergedNode add(@NonNull Node node, @NonNull TemporaryNodeMergeStrategy temporaryNodeMergeStrategy);
 
+    /**
+     * Adds a collection of nodes.
+     *
+     * @param nodes the collection of nodes to add
+     * @since 5.4
+     */
+    void addAll(@NonNull Iterable<? extends Node> nodes);
+
+    /**
+     * Adds the nodes from another node map.
+     *
+     * <p>Effectively calls {@link #addAll(Iterable)} with the result from
+     * {@link #toCollection()}.</p>
+     *
+     * @param nodeMap the node map to merge into this one
+     * @since 5.4
+     */
+    void addAll(@NonNull NodeMap nodeMap);
```

Comment:
I think the first one covers this sufficiently, users wanting to add a `NodeMap` can call `.toCollection()` themselves.

Reply:
Fair enough I guess